doc/source/vol-cli.rst

```diff
@@ -67,12 +67,12 @@ Options
 -s SYMBOL_DIRS, --symbol-dirs SYMBOL_DIRS
     SYMBOL_DIRS is a semi-colon separated list of paths that contain symbol
     files or symbol zip packs.  Symbols must be within a particular directory
-    structure if they depending on the operating system of the symbols,
+    structure if they depend on the operating system of the symbols,
     whilst symbol packs must be in the root of the directory and named after
-    the after the operating system to which they apply.
+    the operating system to which they apply.
 
 -v, --verbose
-    A flag which can be used multiple times, each time increasing the level of
+    A flag which can be used multiple times (up to six, -vvvvvv), each time increasing the level of
```

Comment:
Well, whaddya know.  I always thought all this time it was 7, but I did the maths and you're right, six is the most!  5:D